plugins/lsp/default.nix

```diff
@@ -79,7 +79,6 @@ in {
         "pylsp"
         "pylyzer"
         "pyright"
-        "rnix-lsp"
```

Comment:
Don't include this line, it is needed for the deprecation

Reply:
I am quite noob wrt github, so Iincluded the changes in a new commit. That line is not present in that commit.

Reply:
Ah, gotcha.